hassil/string_matcher.py

```diff
@@ -43,9 +43,9 @@
 )
 
 INTEGER_START = re.compile(r"^(\s*-?[0-9]+)")
-FLOAT_START = re.compile(r"^(\s*-?[0-9]+(?:\.[0-9]+)?)")
+FLOAT_START = re.compile(r"^(\s*-?[0-9]+(?:[.,][0-9]+)?)")
```

Comment:
```suggestion
FLOAT_START = re.compile(r"^(\s*-?[0-9]+(?:[\.,][0-9]+)?)")
```
`.` matches any character (_except line breaks_) - so it still needs to be escaped when the dot itself i meant to be matched
cc @synesthesiam